vllm/compilation/collective_fusion.py

```diff
@@ -41,11 +45,8 @@
 
 logger = init_logger(__name__)
 
-ALLREDUCE_OP = torch.ops.vllm.all_reduce.default
-RMS_OP = torch.ops._C.rms_norm.default
-RMS_ADD_OP = torch.ops._C.fused_add_rms_norm.default
-STATIC_FP8_QUANT_OP = torch.ops._C.static_scaled_fp8_quant.default
-STATIC_FP4_QUANT_OP = torch.ops._C.scaled_fp4_quant.default
+if hasattr(torch.ops._C, "scaled_fp4_quant"):
+    STATIC_FP4_QUANT_OP = torch.ops._C.scaled_fp4_quant.default
```

Comment:
Why did this work before and not now? Should we change how this is registered?

Reply:
Oh yeah I think the registration might have been fixed.

Reply:
I think I'll punt this to a follow-up PR, in general these ops should be cleaned up